### PR TITLE
(feat) Add outpatient clinical encounter views for social and medical history

### PR DESCRIPTION
**Pull request by njorocs**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
ClinicalEncounter

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-clinical-view-app/src/clinical-encounter/clinical-encounter-dashboard-meta.tsx [hunk 1]
```diff
@@ -7,17 +7,8 @@ export const clinicalEncounterNavGroup = {
 export const inPatientClinicalEncounterDashboardMeta = {
   slot: 'patient-chart-in-patient-clinical-encounter-slot',
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/clinical-encounter/clinical-encounter-dashboard-meta.tsx:8]: We should rename it to `patient-chart-clinical-encounter-slot` and also update the path to reflect the change

File: packages/esm-patient-clinical-view-app/src/clinical-encounter/summary/out-patient-summary/patient-medical-history.component.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { formatDate, parseDate, useConfig } from '@openmrs/esm-framework';
+import {
+  Surgical_History_UUID,
+  Accident_Trauma_UUID,
+  Blood_Transfusion_UUID,
+  Chronic_Disease_UUID,
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/clinical-encounter/summary/out-patient-summary/patient-medical-history.component.tsx:8]: I would recomment to have this in snake case and upper case e.g 
```suggestion
  SURGICAL_HISTORY_UUID,
  ACCIDENT_TRAUMA_UUID,
  BLOOD_TRANSFUSION_UUID,
  CHRONIC_DISEASE_UUID,
```

File: packages/esm-patient-clinical-view-app/src/hooks/useClinicalEncounter.ts [hunk 3]
```diff
@@ -12,7 +12,8 @@ export function useClinicalEncounter(patientUuid: string, encounterType: string)
     openmrsFetch,
   );
   const clinicalEncounter = data?.data?.results?.filter((enc) => enc.form.uuid === ClinicalEncounterFormUuid);
-
+  // eslint-disable-next-line no-console
+  console.log('=====clinicalEncounter==', clinicalEncounter);
```
Comment (donaldkibet) [inline packages/esm-patient-clinical-view-app/src/hooks/useClinicalEncounter.ts:16]:
```suggestion
  
```

**Review by donaldkibet — APPROVED** [review]
Thanks @njorocs LGTM 🚀 , left some suggestions.

File: packages/esm-patient-clinical-view-app/src/clinical-encounter/summary/out-patient-summary/patient-social-history.component.tsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    return (
+      <EmptyState
+        displayText={t('socialHistory', 'Social History')}
+        headerTitle={t('socialHistory', 'Social History')}
+        launchForm={handleOpenOrEditClinicalEncounterForm}
+      />
+    );
+  }
+  // eslint-disable-next-line no-console
+  console.log('----------tableRows', tableRows);
```
Comment (makombe) [inline packages/esm-patient-clinical-view-app/src/clinical-encounter/summary/out-patient-summary/patient-social-history.component.tsx]: I hope this log has also been removed

**Comment by gabriel090** [comment]
Yes it was removed

**Pull request merged**